src/App.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
What is app for if it isn't used? (It doesn't get loaded in).

Also the firebase config should only exist in one file

Reply:
definitely. I have made the required change; should App.css and App.test.tsx be removed from the main branch as well?

Reply:
No those are good